css/components/product-listing.css

```diff
@@ -61,6 +64,7 @@
   border: none;
   right: 10px;
 }
+
 .heart-btn .fa-heart {
   color: red;
```

Comment:
you should use hex or rgb() values for setting color

Reply:
Updated

Reply:
After updated code works good